packages/@icon-magic/distribute/README.md

```diff
@@ -1,104 +1,86 @@
 # @icon-magic/distribute
 
-This package is responsible for generating the flavors of the icon in all the
-different types in which it can be consumed. At the end of the build step, we
-have each flavor in it's .svg format, but after generate, we will have more
-optimized .svg files, .png files that can be consumed by iOS and .webp files that
-can be consumed by android.
-
-Generate transforms the set of .svg flavors to their types by running a set of
-plugins based on the type in which we want the output. For example, we can have
-a different set of plugins to obtain the optimized svg and a different set to
-get a .png "type".
-
-After generate has applied all the plugins based on type, we now get flavors
-with types that contain paths to the newly created .type asset. Generate also
-updates the icon config with the newly generated types. An example flavor in the
-icon config (iconrc.json) after it's been through the generate step will look like
-this:
-
-```typescript
-{
-    "name": "filled-24x24",
-    "path": "./filled.svg",
-    "types": {
-        "png": {
-            "name": "filled-24x24",
-            "path": "./filled-24x24.png"
-        },
-        "webp": {
-            "name": "filled-24x24",
-            "path": "./filled-24x24.webp"
-        }
-    }
-},
-```
+This package is responsible for massaging the assets into type specific structures and formats. At the end of the `generate` step, we have optimized .svg files, .png files and .webp files for web, ios and android consumption. You can look in `test/fixtures/input` for an example of the file structure after the generate step. `distribute` organizes and structures those files how they need to be consumed and creates the necessary files for platform consumption.
```

Comment:
You can add a link to the file in the readme. I believe the syntax is `[I'm a relative reference to a repository file](../blob/master/LICENSE)`